opencti-platform/opencti-front/src/private/components/arsenal/vulnerabilities/VulnerabilitySoftwares.js

```diff
@@ -66,7 +66,7 @@ class VulnerabilitySoftwaresComponent extends Component {
           {t('Affected software')}
         </Typography>
         <AddSoftwares
-          vulnerabilityId={vulnerability.id}
+          vulnerability={vulnerability}
           vulnerabilitySoftwares={vulnerability.softwares.edges}
```

Comment:
why don't you just pass only vulnerability as a parameter as the softwares array is embedded?

Reply:
You're right, but I didn't want to change more additional files/components